Sample Applications/DataBindingDemo/AddProductWindow.xaml

```diff
@@ -84,9 +84,7 @@
 
                     <TextBlock Grid.Row="4" Grid.Column="0" Style="{StaticResource SmallTitleStyle}" Margin="0,5,0,5">Category:</TextBlock>
                     <ComboBox Name="CategoryEntryForm" AutomationProperties.Name="Category" Grid.Row="4" Grid.Column="1"
-                              SelectedValue="{Binding Path=Category}"
-                              Style="{StaticResource ComboBoxStyle}"
-                              ItemContainerStyle="{StaticResource ComboBoxItemStyle}" Margin="8,5,0,5">
+                              SelectedValue="{Binding Path=Category}" Margin="8,5,0,5">
```

Comment:
Can you explain this change?

Reply:
It was a miss during testing, thanks for noticing. I have put the style back with triggers for high contrast